changelogs/unreleased/4846-Vishal-Chdhry-minor.md

```diff
@@ -0,0 +1 @@
+ExtensionServices are updated continuously by contour
```

Comment:
Let's also rename this file to `4846-Vishal-Chdhry-small.md`, that seems more appropriate for this change.